.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
```

Comment:
Want to just run this on PR? I've got a separate CI pipeline running on GCP that deploys it to Cloud Run

Reply:
Done